src/Character/Character.jsx

```diff
@@ -43,10 +86,43 @@ export default function Character() {
     return attributeList;
   };
 
+  const renderClassOptions = () => {
+    const classList = [];
+    const classNames = Object.keys(classes);
+    for (let cName of classNames) {
+      classList.push(<option key={sha256(cName)}>{capitalize(cName)}</option>);
+    }
+
+    return classList;
+  };
+
+  const toggleEditMode = () => {
+    setEditMode(!editMode);
+  };
+
+  const updateClass = (e) => {
+    const newClassType = e.target.value;
+    const classStats = classes[newClassType.toLowerCase()];
+    const newAttributes = classStats.stats.attributes;
+    const updatedAttributes = {};
+
+    for (let attr in newAttributes) {
```

Comment:
Can you not just assign this directly by coping the object?

```
const classAtributes = { ...classStats?.stats?.attributes };

setAttributes(classAttributers);
```

Reply:
Totally works, and much cleaner looking.